pkg/apis/core/types.go

```diff
@@ -26,6 +26,11 @@ const (
 	//
 	// If this annotation exists, the system will maintain the annotation value.
 	LogicalClusterPathAnnotationKey = "kcp.io/path"
+
+	// ReplicateAnnotationKey is the annotation key used to indicate that a ClusterRole should be replicated.
```

Comment:
could you please remind me why it is a list?

Reply:
there can be multiple controllers that contribute to this annotation. E.g. from another Api group.